module/Entra/Microsoft.Entra/DirectoryManagement/Set-EntraDirSyncFeature.ps1

```diff
@@ -17,19 +32,27 @@ function Set-EntraDirSyncFeature {
 
         [switch] $Force
     )
+
     PROCESS {
     
         $params = @{}
+        $featuresCollection = @{}
         $customHeaders = New-EntraCustomHeaders -Command $MyInvocation.MyCommand
         if ($PSBoundParameters.ContainsKey("Verbose")) {
             $params["Verbose"] = $PSBoundParameters["Verbose"]
         }
-        if ($null -ne $PSBoundParameters["Feature"]) {
-            $Feature = $PSBoundParameters["Feature"] + "Enabled"
-        }
         if ($null -ne $PSBoundParameters["Enabled"]) {
             $Enabled = $PSBoundParameters["Enabled"]
         }
+        if ($PSBoundParameters.ContainsKey("Feature")) {
+            foreach ($feature in $Features) {
```

Comment:
Confirm that if you pass a single string as parameter value, this code will still work